src/requirements.txt

```diff
@@ -1,10 +1,11 @@
-psycopg2==2.7.6.1
-django==2.1.3
-djangorestframework==3.9.0
-django_filter==2.0.0
-django_extensions==2.1.3
+django-cors-headers==3.0.2
 django-s3-storage==0.12.4
+django==2.2.3
+django_extensions==2.1.9
+django_filter==2.2.0
+djangorestframework==3.10.1
+drf-yasg==1.16.1
 geojson==2.4.1
-drf-yasg==1.11.0
-zappa==0.47.0
-django-cors-headers==3.0.2
+packaging
```

Comment:
is there no version for this?

Reply:
Doesn't appear they're going to use one; see comments here:
https://github.com/axnsan12/drf-yasg/commit/e6219ab8b70cec52dc4cbae04b7a1f315eab504d
And I'll remove it from our requirements once they add it to theirs.